pkg/cli/admin/node/logs.go

```diff
@@ -54,6 +53,9 @@ var (
 
 		# Display cron log file from all masters
 		oc adm node-logs --role master --path=cron
+
+		# Display kubelet log from worker nodes using the new query sub-command
+		oc adm node-logs --role worker --query=kubelet
```

Comment:
Now , we can't support query, please remove this example.

Reply:
Sorry about that. I have now removed it.